### ISSUE TITLE
 Consider renaming `system.network.dropped` to `system.network.packet.dropped` 

### ISSUE THREAD

**Issue by lmolkova** [open-telemetry/semantic-conventions#1700]
See https://github.com/open-telemetry/semantic-conventions/pull/1694#discussion_r1889484257
It's not clear from the metric name what was dropped, would be great to clarify. 

If it turns out that we'd like to keep `system.network.dropped` or rename it to something else, this issue should track updating naming guidelines.

**Comment by braydonk** [comment]
I agree with this! I was actually looking at my notes before the holidays and found this that I wrote before this issue had even opened:

```
`system.network.packets`, should the `dropped` and `packet` metrics be merged to be `system.network.packets` and `system.network.packets.dropped`?
```

Arguably this could even be `system.network.packets.received` so that the pattern matches `system.network.packets.dropped`. Does that sound reasonable?

Side note, I wonder if this (similar to other `system.network` namespace stuff) should simply be moved to `network` as well. For packets received and dropped, I don't feel the `system` namespace improves the understanding or organization of these metrics.

**Comment by mx-psi** [comment]
Discussed on the 2025-02-13. We agree on renaming to `network.packet.received` and `network.packet.dropped` because:
1. The [namespace should be singular](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/naming.md#pluralization)
2. The suffix `received`/`dropped` is more explicit as to what the metric means
3. These metrics could be used in containers as well so we don't think the `system` namespace is needed/correct

**Comment by rogercoll** [comment]
> Arguably this could even be system.network.packets.received so that the pattern matches system.network.packets.dropped. Does that sound reasonable?

Note that the `system.network.packets` metric does not only refer to the "received" packets, but it uses the [network.io.direction attribute](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/registry/attributes/network.md#network-io-direction) to indicate the direction (received or transmit). My proposal would be using the `system.network.io` (similar to `system.disk.io`), which is already defined: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/system-metrics.md#metric-systemnetworkio  (duplicated metric? 🤔 ?

> 3. These metrics could be used in containers as well so we don't think the system namespace is needed/correct

Is this still valid of today? The decision taken in https://github.com/open-telemetry/semantic-conventions/pull/2295 was to keep the `system/container/kubernetes` namespace to make it consistent with the other metrics (disk/network). Also, to denote area-wide metrics.

**Comment by mx-psi** [comment]
> Is this still valid of today? The decision taken in https://github.com/open-telemetry/semantic-conventions/pull/2295 was to keep the system/container/kubernetes namespace to make it consistent with the other metrics (disk/network). Also, to denote area-wide metrics.

I think it is not valid anymore
